bad_connector_woocommerce/models/woo_backend/common.py

```diff
@@ -179,6 +180,44 @@ def _get_stock_field_id(self):
     woo_setting_id = fields.Many2one(comodel_name="woo.settings")
     stock_update = fields.Boolean(related="woo_setting_id.stock_update")
     recompute_qty_step = fields.Integer(string="Recompute Quantity Batch", default=1000)
+    access_token = fields.Char(string="Access Token", readonly=True)
```

Comment:
Don't we need test_access_token field to handle the test mode? your suggestion please Ruchir